cmd/zetaclientd/start.go

```diff
@@ -253,6 +256,14 @@ func Start(_ *cobra.Command, _ []string) error {
 			time.Sleep(30 * time.Second)
 		}
 	}()
+	// pprof http server
+	// zetacored/cometbft is already listening for pprof on 6060 (by default)
+	go func() {
```

Comment:
Consider using `bg.Work` and `bg.WithLogger()`, and `startLogger`

Reply:
I'm not sure that want to change the style of this whole file in this PR.